bindings/matrix-sdk-crypto-ffi/src/backup_recovery_key.rs

```diff
@@ -22,6 +22,9 @@ pub struct BackupRecoveryKey {
 #[derive(Debug, Error, uniffi::Error)]
 #[uniffi(flat_error)]
 pub enum PkDecryptionError {
+    /// Invalid format
```

Comment:
That's a pretty undescriptive error, I guess it's not that important to fix this since it should be gone once the Ruma support has been improved.